### PR TITLE
page userRanking.html.twig userManager userController Connection

### PR DESCRIPTION
**Pull request by serhanif59**

### CODE REVIEW

File: src/View/User/userRanking.html.twig [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+				<th>Rang</th>
+				<th>Pseudo</th>
+				<th>Points</th>
+			</tr>
+			{% set rank = 1 %}
+			{% for user in users %}
+				<tr>
+					<td><img src="/assets/images/{{user.avatar}}"/></td>
+					<td>{{rank}}</td>
+					<td class="left-justify">{{user.name}}</td>
```
Comment (marcosigis) [inline src/View/User/userRanking.html.twig]: in the updated database it was nickName right? just need to change this so it display the nickname

**Review by marcosigis — APPROVED** [review]
just need to change user.name to user.nickName to correctly show the pseudo

**Review by marcosigis — APPROVED** [review]

File: public/assets/js/header.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
-console.log('blop')
-const burgerIcon = document.getElementById("burger")
-const navbar = document.getElementById("mytopnav")
+console.log("blop");
```
Comment (Benja-san) [inline public/assets/js/header.js:1]: I assume console.log is for debugging, could you remove it ?

File: src/View/User/index.html.twig [hunk 3]
```diff
@@ -1,17 +1,73 @@
...
-{% endblock %}
\ No newline at end of file
+	<main>
+		<h1>Classement</h1>
+		<section class="info">
+			<div class="info-session">
+				<h2>Tes infos</h2>
+				<ul>
+					{% set sessionId = 2 %}
+					<! pour le moment sans Session !>
```
Comment (Benja-san) [inline src/View/User/index.html.twig]: please use commenting symfony expressions {#...#}

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**